css/style.css

```diff
@@ -1,19 +1,148 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+/* BACKGROUND IMAGE, TEXT AND BUTTON */
+.page-top-half {
+    position: relative;
+}
+
+.page-top-half .large-text {
+    position: absolute;
+    z-index: 999;
+    margin: 0 auto;
+    left: 0;
+    right: 0; 
+    color: whitesmoke;
+    text-align: center; 
+}
+
+.page-top-half h1 {
+    display: block;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+    font-size: 60px;
+    top: 34%;
+    font-weight: 100;
+}
+
+.page-top-half h2 {
+    font-size: 34px;
+    top: 45%;
+    font-weight: 100;
+}
+
+.background-image {
+    width: 100%;
+} 
+
+button {
+    padding: 15px 40px;
+    background-color: #eb6728;
+    color: white;
+    border: none;
+    border-radius: 4px;
+}
+
+.page-top-half button {
+    position: absolute;
+    left: 46%;
+    top: 59%;
+}
+
+/* BOTTOM HALF OF PAGE STYLING */
+.page-bottom-half {
+    margin-top: 150px;
+    margin-bottom: 160px;
+    text-align: center;
+}
+
+.page-bottom-half h1 {
+    display: block;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+    font-size: 60px;
+    font-weight: 300;
+    color: #3c3b3b;
+}
+
+.icon-container {
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.icon {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.icon .img {
+    max-width: 200px;   
+}
+
+.icon .txt {
+    font-size: 35px;
+    color: #3c3b3b;
+}
+
+/* EXTENDED HOMEPAGE STYLING */
+.extended-feature {
+    background-color: #f7ece8;
+    display: flex; 
+    align-items: center;
+    margin-bottom: 120px; 
+}
 
+.feature-img {
+    width: 840px;
+    padding-right: 250px;
```

Comment:
do you need this padding? Using large fixed `px` values like this can mean things don't display correctly on some screen sizes, so on my screen the quote is pushed too far to the right